SMARTAGRIFOOD/vineyardData/schema.json

```diff
@@ -1,165 +1,227 @@
-        "enum": [
-          "freeForm",
-          "rows"
-        ],
-        "type": "string"
-      },
-      "owner": {
-        "description": "Relationship. A List containing a JSON encoded sequence of characters referencing the unique Ids of the owner(s)",
-        "items": {
-          "description": "Property. Unique identifier of the entity"
-        },
-        "type": "array"
-      },
-      "relatedSource": {
-        "description": "Relationship. List of IDs the current entity may have in external applications",
-        "items": [
-          {
-            "type": "object",
-            "values": {
-              "application": {
-                "description": "Property. Unique identifier of the entity"
-              },
-              "applicationEntityId": {
-                "type": "string"
-              }
-            }
-          }
-        ],
-        "type": "array"
-      },
-      "seeAlso": {
-        "description": "Relationship. List of uri pointing to additional resources about the item",
-        "oneOf": [
-          {
-            "items": {
-              "format": "uri",
-              "type": "string"
-            },
-            "minItems": 1,
-            "type": "array"
-          },
-          {
-            "format": "uri",
-            "type": "string"
-          }
-        ]
-      },
-      "source": {
-        "description": "Property. A sequence of characters giving the original source of the entity data as a URL. Recommended to be the fully qualified domain name of the source provider, or the URL to the source object.",
-        "type": "string"
-      },
-      "type": {
-        "description": "Property. NGSI Entity Type: It has to be AgriVineyardBlock",
-        "enum": [
-          "AgriVineyardBlock"
-        ],
-        "type": "string"
-      }
-    }
-    }
-    ],
-    "required": [
-        "id",
-        "type"
-    ]
-
-}
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$schemaVersion": "0.0.5",
+    "modelTags": "",
+    "$id": "https://github.com/smart-data-models/incubated/SMARTAGRIFOOD/vineyardData/schema.json",
+    "title": "Smart Data Models - Vineyard Block",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/GSMA-Commons"
+        },
+        {
+            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+        },
+        {
+            "$ref": "https://smart-data-models.github.io/dataModel.Agrifood/agrifood-schema.json#/definitions/AgriFood-Commons"
+        },
+        {
+        "description": "This entity contains a harmonised description of a vineyard block (uniform zone of the vineyard). ",
+        "properties": {
+          "agroVocConcept": {
+            "description": "Relationship. Reference to the agrovoc term associated with this item",

```

Comment:
if it is a relationship needs to include the EntityIdentifierType from the https://github.com/smart-data-models/data-models/blob/master/common-schema.json